index.html

```diff
@@ -151,17 +151,15 @@ <h3 class="fw-bold mt-3 subline ms-3">Remodelación</h3>
 			<!-- Sobre nosotros -->
 			<section style="background-color: white;">
 				<div class="p-5">
-					<h2><strong class="fw-bold text-color-main h1">PROFESIONALES</strong> PARA TI</h2>
-					<h2 class="line-h">SOBRE NOSOTROS</h2>
 					<div class="mx-5">
-						<p class="fst-italic mt-5 h5" style="text-indent: 2rem;">Somos Carpitec43 una empresa líder en carpinteria general. Fundada en 1968
-							con una trayectoria de 53 años, encargados de llevarte siempre lo mejor en
-						materiales y diseño, a la vanguardia con los standares y equipos actuales. Ubicada en </p>
-						<blockquote class="mt-5"><p class="fst-italic fw-bold">
+						<h1 class="text-start"><strong class="fw-bold text-color-main h2">PROFESIONALES</strong> PARA TI </h1>
+						<h3 class="line-h">SOBRE NOSOTROS</h3>
+						<p class="fst-italic mt-4 h5"class="text-start">Somos Carpitec43 una empresa líder en carpinteria general. Fundada en 1968 con una trayectoria de 53 años, encargados en llevarte siempre lo mejor en materiales y diseño, a la vanguardia con los standares y equipos actuales, entregando siempre cada proyecto con el mayor cariño posible para dar el mejor resultado final. Puedes ubicarnos en el Galpón No. 7 piso 2 ,Calle San Julian, Av. Libertador. </p>
```

Comment:
Ojo con las clases extra
```suggestion
						<p class="fst-italic mt-4 h5">Somos Carpitec43 una empresa líder en carpinteria general. Fundada en 1968 con una trayectoria de 53 años, encargados en llevarte siempre lo mejor en materiales y diseño, a la vanguardia con los standares y equipos actuales, entregando siempre cada proyecto con el mayor cariño posible para dar el mejor resultado final. Puedes ubicarnos en el Galpón No. 7 piso 2 ,Calle San Julian, Av. Libertador. </p>
```